modules/apps/expando/expando-service/service.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.4.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_4_0.dtd">
 
-<service-builder auto-import-default-references="false" change-tracking-enabled="true" mvcc-enabled="true" package-path="com.liferay.expando">
+<service-builder auto-import-default-references="false" change-tracking-enabled="true" dependency-injector="ds" mvcc-enabled="true" package-path="com.liferay.expando">
 	<namespace>Expando</namespace>
 	<entity local-service="true" name="ExpandoColumn" remote-service="true">
```

Comment:
@slnn, based on the doc from liferay-service-builder_7_4_0.dtd, the module is new. 

> For new OSGi modules, the default value is "ds"

It seems the commit is not necessary. Please verify this again. If it is necessary, I suggest we should provide the related explanation in the commit message.